os/drivers/kernel_test/Make.defs

```diff
@@ -19,7 +19,11 @@
 include kernel_test$(DELIM)kernel$(DELIM)Make.defs
 include kernel_test$(DELIM)network$(DELIM)Make.defs
 
+ifeq ($(CONFIG_DRIVERS_KERNEL_TEST),y)
 CSRCS += kernel_test_drv.c
+else ifeq ($(CONFIG_TC_NET_PBUF),y)
+CSRCS += kernel_test_drv.c
```

Comment:
CONFIG_DRIVERS_KERNEL_TEST and CONFIG_TC_NET_PBUF are different layer. so this code seems not good to me. You'd better to add dependency between both config in Kconfig. Please check 'depends on' and 'select' command in Kconfig

Reply:
changed. check now